multigeiger/webconf.cpp

```diff
@@ -80,6 +120,25 @@ void setup_webconf() {
   iotWebConf.getWifiSsidParameter()->label = "WiFi client SSID";
   iotWebConf.getWifiPasswordParameter()->label = "WiFi client password";
 
+  // add the setting parameter
```

Comment:
maybe rather omit such comments, that's quite obvious from the code already.

Reply:
Better more than too little comment ...

Reply:
no, it is not better if it does not add information - you have to read all the stuff, it produces longer source code.

sometimes it also has typos or sounds strange...